src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        String dateFrom = "01.04.2019";
```

Comment:
Any magic numbers should be constants. Consider defining 'dateFrom' and 'dateTo' as constant fields if they are not meant to be changed.